### PR TITLE
Update POST-Properties-Reference.md

### PR DESCRIPTION
**Pull request by Aaron-Gor**
@lsnovich

### CODE REVIEW

File: POST-Properties-Reference.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 POST/meal/lunch/main/burger/burger properties 
 
-The customer can order one or more beefburger or a veggie burger. The size can be 300 gr or 400 gr. The customer has the option to specify how the burger is cooked: medium rare, medium or well-done. The customer has the option to choose the bun type: white or whole meat. The customer can choose the type of topping for the burger: lettuce, tomato, onion or no topping. The customer can choose which condiment to have with the burger: ketchup, BBQ sauce, mushroom sauce or no condiment.
+The customer can order one or more beefburger or a veggie burger. The size can be 300 gr or 400 gr. The customer has the option to specify how the burger is cooked: medium rare, medium or well-done.
```
Comment (lsnovich) [inline POST-Properties-Reference.md:5]:
```suggestion
The customer can order one or more beef or veggie burgers. The size can be 300 or 400 g. The customer has the option to specify how the burger is cooked: medium rare, medium, or well-done. 
```

**Comment by Aaron-Gor** [comment]
@lsnovich Fixed #7

File: POST-Properties-Reference.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 POST/meal/lunch/main/burger/burger properties 
 
-The customer can order one or more beefburger or a veggie burger. The size can be 300 gr or 400 gr. The customer has the option to specify how the burger is cooked: medium rare, medium or well-done. The customer has the option to choose the bun type: white or whole meat. The customer can choose the type of topping for the burger: lettuce, tomato, onion or no topping. The customer can choose which condiment to have with the burger: ketchup, BBQ sauce, mushroom sauce or no condiment.
+The customer can order one or more beefburger or a veggie burger. The size can be 300 gr or 400 gr. The customer has the option to specify how the burger is cooked: medium rare, medium or well-done. 
+The customer has the option to choose the bun type: white or whole meat. The customer can choose the type of topping for the burger: lettuce, tomato, onion or no topping. The customer can choose which condiment to have with the burger: ketchup, BBQ sauce, mushroom sauce or no condiment.
```
Comment (lsnovich) [inline POST-Properties-Reference.md:6]:
```suggestion
The customer has the option to choose the bun type: white or whole wheat. The customer can choose the type of topping for the burger: lettuce, tomato, onion, or no topping. The customer can choose which condiment to have with the burger: ketchup, BBQ sauce, mushroom sauce, or no condiment.
```

**Review by lsnovich — CHANGES_REQUESTED** [review]
2 changes - to accept please click Accept commit / accept changes

**Pull request closed without merging**